From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 27 Jan 2021 23:14:47 +0000
Subject: [PATCH] Fix build when lame is disabled

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 mscore/exportdialog.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mscore/exportdialog.cpp b/mscore/exportdialog.cpp
index 7d4a3b1..45c8e8a 100644
--- a/mscore/exportdialog.cpp
+++ b/mscore/exportdialog.cpp
@@ -76,7 +76,7 @@ ExportDialog::ExportDialog(Score* s, QWidget* parent)
 # ifndef USE_LAME
       // Disable .mp3 option if unavailable
       QStandardItem* mp3Item = fileTypeComboBoxModel->item(3);
-      mp3Item->setFlags(audioItem->flags() & ~Qt::ItemIsEnabled);
+      mp3Item->setFlags(mp3Item->flags() & ~Qt::ItemIsEnabled);
 # endif
 # ifndef HAS_AUDIOFILE
       // Disable .wav, .flac and .ogg options if unavailable
