From 10e69feac7192a28d54da411275b3814cfbf9cd3 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Mon, 8 Apr 2019 15:13:31 -0700
Subject: [PATCH] Fix build when LAME is disabled

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 mscore/musescore.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mscore/musescore.cpp b/mscore/musescore.cpp
index 584dd94..c972622 100644
--- a/mscore/musescore.cpp
+++ b/mscore/musescore.cpp
@@ -6653,7 +6653,6 @@ bool MuseScore::saveMp3(Score* score, QIODevice* device, bool& wasCanceled)
       {
 #ifndef USE_LAME
       Q_UNUSED(score);
-      Q_UNUSED(name);
       return false;
 #else
       EventMap events;
-- 
2.21.0

